### ISSUE TITLE
Accessibility: Text field control doesn't adapt to high contrast black/White theme of windows.

### ISSUE THREAD

**Issue by deepgu** [wixtoolset/issues#5959]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?
3.7.1224.0

* Which version of Visual Studio are you building with (if any)?
N/A

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

N/A

* Which version of .NET are you building with?

N/A

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

N/A

* Describe the problem and the steps to reproduce it.

Disabled text control in installer GUI is not being highlighted (in green border) in high contrast windows theme, but password controls works correctly. But configuration for both elements is same.

![image](https://user-images.githubusercontent.com/32868409/53210781-e5c92300-35f3-11e9-8ff1-f1f1f19b14b1.png)

* Describe the behavior you expected and how it differed from the actual behavior.

Disabled text control should be highlighted.

**Comment by robmen** [comment]
Yeah. Windows Installer team should fix that in the MSI engine. Rendering of actual controls in the MSI UI is controlled by the MSI engine.
